asian-cuisine.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
Let's create a commit for each page next time :)

Reply:
Thanks Robert!